src/entities/applet/model/hooks/useStartEntity.ts

```diff
@@ -23,6 +23,15 @@ export const useStartEntity = () => {
     )
   }
 
+  function removeActivityProgress(activityId: string, eventId: string): void {
+    dispatch(
```

Comment:
I don't think it is necessary to delete activity/flow before starting a new one. The implementations of `flowStarted` and `activityStarted` reducers already imply creating a new object that is going to replace the previous one:
<img width="584" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-web-refactor/assets/20073193/278a3c53-70e8-4a9e-8e8f-7a3480ca99cc">

Reply:
It turned out that removeActivityProgress removes the state of in-progress activities, like step, answers, etc